docs/feature-store/feature-sets.md

```diff
@@ -171,9 +171,24 @@ When defining a source, it maps to nuclio event triggers. <br>
 Note that you can also create a custom `source` to access various databases or data sources.
 
 ### Target stores
-By default the feature sets are stored as both parquet file for training and as a key value table (in the Iguazio MLOps platform) for online serving. <br>
-The parquet file is ideal for fetching large set of data for training while the key value is ideal for an online application since it supports low latency data retrieval based on key access. <br>
+By default, the feature sets are stored as both parquet file for training and as a key value table (in the Iguazio MLOps platform or a Redis DB) for online serving. <br>
+The parquet file is ideal for fetching large set of data for training while the key value is ideal for an online application since it supports low latency data retrieval based on key access. 
 
 ```{admonition} Note
 When working with the Iguazio MLOps platform the default feature set storage location is under the "Projects" container: <project name>/fs/.. folder. 
 The default location can be modified in mlrun config or specified per ingest operation. The parquet/csv files can be stored in NFS, S3, Azure blob storage and on Iguazio DB/FS.
+```
+#### Redis target store
+The Redis online target is called, in MLRun, `RedisNoSqlTarget`. RedisNoSqlTarget and NoSqlTarget have identical functionality 
```

Comment:
=>  RedisNoSqlTarget and NoSqlTarget (the Iguazio NoSql DB) have identical functionality